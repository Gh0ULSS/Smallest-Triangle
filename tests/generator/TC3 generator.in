-N=7 -mindist=2 -rseed=24