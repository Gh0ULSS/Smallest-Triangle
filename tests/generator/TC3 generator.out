21.23, 33.98
-31.74, 49.83
-30.59, 17.09
-40.82, 25.77
-34.87, 20.66
22.44, 26.59
-5.86, 36.58
